qa/sc_versions_and_mc_certs.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python3
+
+import codecs
+import pprint
+import time
+from decimal import Decimal
+
+from SidechainTestFramework.sc_boostrap_info import SCNodeConfiguration, SCCreationInfo, MCConnectionInfo, \
+    SCNetworkConfiguration
+from SidechainTestFramework.sc_test_framework import SidechainTestFramework
+from test_framework.mc_test.mc_test import CertTestUtils, generate_random_field_element_hex
+from test_framework.util import assert_false, start_nodes, \
+    websocket_port_by_mc_node_index
+from SidechainTestFramework.scutil import bootstrap_sidechain_nodes, \
+    start_sc_nodes, generate_next_blocks, generate_next_block, \
+    assert_true, create_alien_sidechain, create_certificate_for_alien_sc
+from SidechainTestFramework.sc_forging_util import *
+
+"""
+Configuration:
+    Start 1 MC node and 1 SC node (with default websocket configuration).
+    SC node connected to the first MC node.
+
+Test:
+    TODO    
+"""
+
+
+class SCVersionsAndMCCertificates(SidechainTestFramework):
+
+    sc_nodes_bootstrap_info = None
+    sc_withdrawal_epoch_length = 10
+
+    def setup_nodes(self):
+        num_nodes = 1
+        # Set MC scproofqueuesize to 0 to avoid BatchVerifier processing delays
+        return start_nodes(num_nodes, self.options.tmpdir, extra_args=[['-debug=sc', '-debug=ws',  '-logtimemicros=1', '-scproofqueuesize=0']] * num_nodes)
+
+    def sc_setup_chain(self):
+        mc_node = self.nodes[0]
+        sc_node_configuration = SCNodeConfiguration(
+            MCConnectionInfo(address="ws://{0}:{1}".format(mc_node.hostname, websocket_port_by_mc_node_index(0)))
+        )
+        network = SCNetworkConfiguration(SCCreationInfo(mc_node, 100, self.sc_withdrawal_epoch_length, btr_data_length=0
+                                                        , sc_creation_version=1), sc_node_configuration)
+
+        mc_node.generate(450)
```

Comment:
do we need this? As far as I remember, `bootstrap_sidechain_nodes` will generate 450 blocks to switch to the proper fork with sidechain versions activation.

Reply:
You are right, this is redundant